### PR TITLE
move all cockpit related files to cockpit subpackage

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
this fixes https://github.com/theforeman/foreman-selinux/pull/163#issuecomment-1934018722

File: packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec [hunk 1]
```diff
@@ -129,11 +128,15 @@ install -Dp -m0644 %{buildroot}%{gem_instdir}/extra/cockpit/settings.yml.example
 %config(noreplace) %{_sysconfdir}/foreman/cockpit/cockpit.conf
 %config(noreplace) %{_sysconfdir}/foreman/cockpit/foreman-cockpit-session.yml
 %{_unitdir}/foreman-cockpit.service
+%{gem_instdir}/extra/cockpit
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec:131]: Why do we need the extra directory? All files are copied, except `foreman-cockpit-session` which is a symlink. Perhaps we should simply make line 84 an `install` and `%exclude %{gem_instdir}/extra` instead.

Comment (evgeni) [reply packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec:131]: maybe. I was not planing to re-think the packaging. but I can :)

Comment (evgeni) [reply packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec]: well, let's see how that goes

**Pull request merged**